com.github.alainm23.byte.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "com.github.alainm23.byte",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.alainm23.byte",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--socket=pulseaudio"
```

Comment:
MPRIS doesn't seem to work because you're missing per https://github.com/alainm23/byte/blob/935e240bf6d0802c79289003517528bf141efc20/src/Services/Indicator.vala#L11
```
    "--own-name=org.mpris.MediaPlayer2.Byte",
```
You could also change the dbus name upstream to `org.mpris.MediaPlayer2.com.github.alainm23.byte`